cns/hnsclient/hnsclient_windows.go

```diff
@@ -301,7 +304,16 @@ func createHostNCApipaNetwork(
 		}
 
 		// Create loopback adapter needed for this HNS network
-		if interfaceExists, _ := networkcontainers.InterfaceExists(hostNCLoopbackAdapterName); !interfaceExists {
+		// We need to fitst check the existence of either "LoopbackAdapterHostNCConnectivity" or the vEthernet(LoopbackAdapterHostNCConnectivity) interfaces
+		// If neither exists, we create the loopback adapter with the specified IP configuration.
+		loopbackInterfaceExists, _ := networkcontainers.InterfaceExists(hostNCLoopbackAdapterName)
+		vethernetLoopbackInterfaceExists, _ := networkcontainers.InterfaceExists(vEthernethostNCLoopbackAdapterName)
```

Comment:
Let's check this only if the first check returns false.

Reply:
The Veth can only exist if the HNS add the interface and remove the original loopback.